hack/ci/vagrant-helper.sh

```diff
@@ -22,10 +22,7 @@ if [ ! -f "$SSH_CONFIG" ]; then
   vagrant ssh-config > "$SSH_CONFIG"
 fi
 
-sudo="sudo"
 if [ "$ROOTLESS" = "rootless" ]; then
-  sudo=
+  exec ssh -F "$SSH_CONFIG" default KIND_EXPERIMENTAL_PROVIDER="$KIND_EXPERIMENTAL_PROVIDER" "${@}"
```

Comment:
TIL exec runs and exits

Reply:
exec is like the syscall, it does not fork, it replaces the current process.